.github/workflows/test-build-deploy.yml

```diff
@@ -48,7 +48,7 @@ jobs:
   doc-validator:
     runs-on: ubuntu-latest
     container:
-      image: grafana/doc-validator:latest
+      image: grafana/doc-validator:v1.0.0
```

Comment:
Could you also change it in `check-doc-validator` target in the `Makefile`, please?